### PR TITLE
Task/74 programmatic bottom navigation bar

### PR DESCRIPTION
**Pull request by storkandstars**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -69,6 +70,27 @@ class _SimplioAppState extends State<SimplioApp> {
         ],
         child: MaterialApp(
           title: 'Simplio',
+          scrollBehavior: const ScrollBehavior(
+            androidOverscrollIndicator: AndroidOverscrollIndicator.stretch,
```
Comment (jvrastil) [inline lib/main.dart:74]: This is `deprecated` now:
```
'Use ThemeData.useMaterial3 or override ScrollBehavior.buildOverscrollIndicator. '
      'This feature was deprecated after v2.13.0-0.0.pre.'
```

File: lib/view/routes/guards/auth_guard.dart [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:simplio_app/logic/auth_bloc/auth_bloc.dart';
+
+class AuthGuard extends StatelessWidget {
```
Comment (jvrastil) [inline lib/view/routes/guards/auth_guard.dart:5]: There is still the old file in `view/guards` folder. If it's not needed for some future development please remove the folder together with the old file

File: lib/main.dart (hunk 1 above)
Comment (storkandstars) [reply lib/main.dart:74]: I will remove it for this release. It is not essential to any functionality.

**Review by jvrastil — APPROVED** [review]
looks good now

**Pull request merged**